layers/b2qt/recipes-qt/automotive/qtapplicationmanager/sc-config.yaml

```diff
@@ -6,7 +6,7 @@ formatType: am-configuration
 ---
 containers:
   selection:
-    - com.pelagicore.radio: "softwarecontainer"
```

Comment:
Is this just some example config? Is that why it doesn't matter if it's changed?

Reply:
This is not an example config, but one can decide to run any application inside container. So we can have more applications with "softwarecontainer" value set in this configuration. For PELUX we are just running one of the several applications inside the container. The important thing is that whatever application we decide to run inside a container, the access to resources needed by that application must be set in the service manifest file.